config/locales/views/errors/en.yml

```
@@ -1,7 +1,13 @@
 ---
 en:
   errors:
+    internal_server_error:
+      subtitle: Please try again. If the problem continues, don't hesitate to reach out to us.
```

I would keep the translations closer.
```suggestion
      subtitle: Please try again. If the problem continues, please reach out to us.
```